darts/models/standard_regression_model.py

```diff
@@ -70,3 +70,24 @@ def predict(self, features: List[TimeSeries]):
         super().predict(features)
         y = self.model.predict(self._get_features_matrix_from_series(features))
         return TimeSeries(pd.DataFrame(y, index=features[0].time_index()), self.train_target.freq())
+
+    def backtest(self,
```

Comment:
I don't think it's worth to rewrite the backtest() method here just to add the default `start = self.train_n_points`.

With this model, `self.train_n_points` can be set to `None`, in which case the model will just retrain on whatever is available. I think we could assume that this is the "default", and remove this backtest() implementation. We should however slightly change the `__init__` function so that the argument becomes `train_n_points: Optional[int]=None`.

Reply:
> With this model, self.train_n_points can be set to None, in which case the model will just retrain on whatever is available. I think we could assume that this is the "default", and remove this backtest() implementation.

Indeed, `train_n_points` can totally be made Optional assuming this default behavior.

However I still think that it would be somewhat misleading to silently train on less than `train_n_points` when `train_n_points` has been explicitly specified. What do you think ?

Reply:
That's a good point. Like you suggest, we should probably display a warning when `train_n_points` has been specified and the model is fit on fewer points. Then in `backtest` the warning would also be displayed as it calls the fit() method.